### ISSUE TITLE
Version 1.5.2 has stopped working

### ISSUE THREAD

**Issue by DannyBen** [brunofacca/active-record-query-trace#18]
Hi,

I just noticed now that my logs no longer contain "Called From" rows.
Locking the gem back with `gem 'active_record_query_trace', '1.5'` solves it

This is my initializer, and I know it is called.

```
# config/initializers/activerecord_query_trace.rb
if defined? ActiveRecordQueryTrace
  ActiveRecordQueryTrace.enabled = true
  ActiveRecordQueryTrace.level = :app
  ActiveRecordQueryTrace.ignore_cached_queries = true
  ActiveRecordQueryTrace.lines = 2
end
```

Is there some change I am unaware of in 1.5.2, or is this a bug?

**Comment by ruckus** [comment]
Hi @DannyBen  - I can see how this might've caught people by surprise.

The answer is that in `1.5.2` we changed it from the string "Called From" to the character `↳` - which you should see in your logs. 

Maybe I should replace it with both: `Called from ↳`

**Comment by DannyBen** [comment]
**TL;DR** - Maybe separate the output to a separate log, with customizable format, like the [i18n gem does](https://github.com/fphilipe/i18n-debug#usage)?

---

Oh... wow - I did not notice it!
And since I am using only 2 trace lines, the actual output by the plugin got completely lost in everything else.

The "Called From" was easier to "catch" among all the clutter of the log.

In addition, the character you are using does not show well in some situations (like when viewed in Sublime Text and possibly other editors).

Below are some screenshots, but as for what is the best way to present this "called from" comment, I am unsure. Here are some thoughts for your consideration:

1. I think the "Called From" comment should be relatively easy to spot. Maybe its just me, but its one of the primary reasons I open the development log.
2. In the 1.5 release, the text "Called From" had zero spaces at the beginning of the line, which made it easy to spot on one hand, but a little confusing on the other hand, since it looked like it starts a section, while in reality, it continues the "select" statement above it (see screenshot below).
3. I would like to cast my vote against using a special character. Rails logs are messy enough as it is without introducing new things.
4. It would probably be "good behavior" by the plugin to simply mention that "I am responsible for this log line" - so perhaps, the marker should be something like "Query Trace > " - then its like combining the "Called From" from before, with an ASCII friendly character. Or something...
5. Another, more radical suggestion - have this logged in a **separate log**. I am also using the i18n debug gem, and it lets me send its output (as well as format the output) to a completely separate log. This makes it very easy to debug and makes it much easier to separate signal from noise. I am definitely in favor of this option.

#### This is how the logs look with 1.5
![before](https://cloud.githubusercontent.com/assets/2405099/11847726/8095f280-a427-11e5-96fe-76ab1ad7223a.png)

#### And this is how they look with 1.5.2
![after](https://cloud.githubusercontent.com/assets/2405099/11847727/82e1b420-a427-11e5-87ef-3a17f12950ab.png)


For convenience, below are the same versions with color codes cleaned up:

#### 1.5
![before2](https://cloud.githubusercontent.com/assets/2405099/11847745/959242ec-a427-11e5-8737-25ea85c1c797.png)


#### 1.5.2
![after2](https://cloud.githubusercontent.com/assets/2405099/11847748/988751e0-a427-11e5-8f9f-17a7023293d5.png)

**Comment by ruckus** [comment]
Yeah I agree. The special-character was provided by a Pull Request and I didnt really think it through and I just merged it in. 

I've just released `1.5.3` with ` Query Trace > `

**Comment by DannyBen** [comment]
Ok, cool. Good compromise.
Maybe when I have time, I may try to make it so it reports to a separate log file (optional and configurable). I think it will be perfect.

Thanks for the quick release.

**Comment by brunofacca** [comment]
Closing due to inactivity.
